de4a-connector/src/main/java/eu/de4a/connector/client/Client.java

```diff
@@ -185,25 +192,28 @@ public Object sendEvidenceRequest(RequestTransferEvidenceUSIIMDRType evidenceReq
 			logger.debug("Request: {}", evidenceRequest.getRequestId());
 		}
 		String urlRequest = endpoint + (isUsi ? "USI" : "IM");
+		HttpHeaders headers = new HttpHeaders();
+		headers.setContentType(MediaType.APPLICATION_XML);
 
 		try {
 			if (!isUsi) {
 				RequestExtractEvidenceIMType requestExtractEvidence = MessagesUtils
 						.transformRequestToOwnerIM(evidenceRequest);
 				String reqXML = DE4AMarshaller.doImRequestMarshaller().getAsString(requestExtractEvidence);
-				ResponseEntity<String> response = restTemplate.postForEntity(urlRequest, reqXML, String.class);
-
-				// Transform ResponseExtractEvidence into ResponseTransferEvidence
-				// TODO move to utils class
+				
+				ResponseEntity<String> response = restTemplate.postForEntity(urlRequest, 
+						new HttpEntity<>(reqXML, headers), String.class);
+				
 				ResponseExtractEvidenceType responseExtractEvidenceType = DE4AMarshaller
 						.doImResponseMarshaller(IDE4ACanonicalEvidenceType.NONE)
```

Comment:
Doesn't this need to be some IDE4ACanonicalEvidence.multiple(EDE4ACanonicalEvidence.T41_UC1_2021_02_11, T42_COMPANY_INFO_V06)

Reply:
or something similar

Reply:
The Connector should be abstracted of any canonicalEvidenceType, it only cares about the message compilance